src/utils.js

```diff
@@ -453,29 +453,19 @@ exports.insertElement = function(elm, doc, target) {
   } catch (e) {}
 };
 
-exports.insertPixel = function (url) {
+exports.triggerPixel = function (url) {
   const img = new Image();
-  img.id = _getUniqueIdentifierStr();
   img.src = url;
```

Comment:
my experience is that just creating the image with a src works on all major browsers, but don't have an authoritative source.